arm-sql/2015-05-01-preview/swagger/failoverGroups.json

```diff
@@ -0,0 +1,616 @@
+                        "schema": {
+                            "$ref": "#/definitions/ArmErrorResponseMessage"
+                        }
+                    }
+                },
+                "deprecated": false
+            }
+        },
+        "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/failoverGroups/{failoverGroupName}/failover": {
+            "post": {
+                "tags": [
+                    "FailoverGroups"
+                ],
+                "description": "Starts planned failover operation for the failover group.",
```

Comment:
I assume that like replicationLinks/failover this is called on the new primary, is this correct? If so, please describe. Below is what I wrote in Azure CLI for this operation, consider something similar. 

Command
    az sql db replica set-primary: Sets which replica database is primary by failing over from the
    current primary replica database.

The swagger doc for replicationLinks/failover probably needs to be improved as well.